pkg/arvo/app/chat-hook.hoon

```diff
@@ -637,12 +654,7 @@
   ::  about group path shape, and the chat that would match it.
   ::TODO  remove me at some point.
   ::
-  ?.  .^(? %gu (scot %p our.bol) %metadata-store (scot %da now.bol) ~)
-    ?:  ?=([%'~' @ ^] group-path)
-      ~&  [%assuming-ported-legacy-group group-path]
-      [t.group-path]~
```

Comment:
Is this not still sane to assume? (With change to just `group-path`, of course.) Same below.

Reply:
We don't want to do anything different in the "~" group case than the non-sig group case.